### PR TITLE
Add issuer instance api

### PR DESCRIPTION
**Pull request by aljones15**
Adds a single new endpoint: POST `issuers/:issuerId/credentials`.

1. Takes in a json object with a credential and optional options.
2.Finds the issuer instance using the `req.params.issuerId`
3. signs using that instance the credential
4. packages the credential inside of a VP
5. returns the VP to the client (they will need to set the holder themselves)

### CODE REVIEW

File: lib/http.js [hunk 1]
```diff
@@ -27,6 +28,7 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       instance: '/vc-issuer/instances/:instanceId',
       instanceIssuer: '/vc-issuer/instances/:instanceId/issuer',
       automatedIssue: '/vc-issuer/issue',
+      issueCredential: '/issuers/:issuerId/credentials',
```
Comment (davidlehn) [inline lib/http.js]: Should this line up with the other endpoints?
```suggestion
      issueCredential: '/vc-issuer/instances/:instanceId/credentials',
```

Comment (aljones15) [reply lib/http.js]: it should be the way it is `/issuers/:issuerId/credentials`.  This was decided on this morning in a design meeting.

**Comment by mattcollier** [comment]
@aljones15 while we wait for approvals, could you please update the changelog?

File: lib/http.js (hunk 1 above)
Comment (dlongley) [inline lib/http.js]: `issuerId` here is `instanceId`, please change the name to avoid confusion.

File: lib/http.js [hunk 2]
```diff
@@ -302,4 +304,36 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
...
       res.json(results);
     }));
+  // issue a single VC if the posted data meets business rules.
+  app.post(
+    routes.issueCredential,
+    // TODO: validate body {options, credential}
+    asyncHandler(async (req, res) => {
+
+      // TODO: inspect/validate API token
+      // token will need to align with issuer.id
```
Comment (dlongley) [inline lib/http.js]:
```suggestion
      // token will need to align with req.params.instanceId
```

File: lib/http.js [hunk 3]
```diff
@@ -302,4 +304,36 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
...
+
+      const {
+        // FIXME implement options
+        options = {},
+        credential,
+      } = req.body;
+      const {issuerId} = req.params;
```
Comment (dlongley) [inline lib/http.js]:
```suggestion
      const {instanceId} = req.params;
```

File: lib/http.js [hunk 4]
```diff
@@ -302,4 +304,36 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
...
+      // get instance for issuer
+      const {instance} = await vcIssuer.instances.get({id: issuerId});
```
Comment (dlongley) [inline lib/http.js]:
```suggestion
      const {instance} = await vcIssuer.instances.get({id: instanceId});
```

File: lib/http.js [hunk 5]
```diff
@@ -302,4 +304,36 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
...
+      // TODO: inspect/validate API token
+      // token will need to align with issuer.id
+
+      const {
+        // FIXME implement options
+        options = {},
+        credential,
+      } = req.body;
+      const {issuerId} = req.params;
+      // get instance for issuer
```
Comment (dlongley) [inline lib/http.js]:
```suggestion
```

**Comment by aljones15** [comment]
wait on merging this testing the changes.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,10 @@
 # bedrock-vc-issuer ChangeLog
 
+## 0.1.1 -
```
Comment (davidlehn) [inline CHANGELOG.md]: This package hasn't been released yet.  0.1.0 below is the first version (or renamed as 1.0.0 if that's more appropriate).  Doesn't make much sense to add just one specific entry for the first version that has 100+ commits.

Comment (aljones15) [reply CHANGELOG.md]: change it to `1.0.0` as the version in the real notes?

Comment (aljones15) [reply CHANGELOG.md]: Addressed here: https://github.com/digitalbazaar/bedrock-vc-issuer/pull/27/commits/95fb1ef9f23bb5a0a93ca1d144fe9929c217961b

File: lib/http.js (hunk 1 above)
Comment (aljones15) [reply lib/http.js]: instanceId https://github.com/digitalbazaar/bedrock-vc-issuer/pull/27/commits/b038742b4f4b5161fcba3d7c4da0168b24ef08ac

**Pull request merged**